StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/IntentConfirmParams.swift

```diff
@@ -133,7 +134,11 @@ class IntentConfirmParams {
         } else {
             if isSettingUp {
                 // Checkbox is hidden
```

Comment:
Should we assert that `saveForFutureUseCheckboxState` is `.hidden` here?

Reply:
Sure, i will revisit this in a subsequent PR when hiding the checkbox when payment_method_save is set to disabled.

Reply:
Added assert here:
https://github.com/stripe/stripe-ios/pull/3884